### DISCUSSION TITLE
Connect new blocks

### DISCUSSION THREAD

**Discussion by digitall-it** [rahmanramsi/filament-editorjs#10] [Q&A]
The EditorJS documentation has a tutorial on how to create new blocks. Is then possible to use those with this plugin?

**Comment by rahmanramsi** [comment]
Sorry it is not possible for now, maybe in the future when I have time. Currently busy with another project.
Feel free to open PR if you want to do it

**Reply by sbritz** [reply]
Thx for this awesome filament-plugin!!!! It saves my day, really. 💯 :-)
And yeah, it would be great, if it were possible to create own new blocks.
Keep on going. 👍
